### DISCUSSION TITLE
求游戏《只狼：影逝二度》

### DISCUSSION THREAD

**Discussion by 568614672** [Gamuxorg/bbs#406] [投递和请求游戏]
游戏名称：《只狼：影逝二度》
英文名称：SEKIRO: Shadows Die Twice
如果有的话，请上传一下，谢谢！

**Comment by amorous-monk** [comment]
直接安装Windows版本，用steam运行就可以了。

**Comment by Crany-web** [comment]
已添加，建议使用Motrix下载
